src/lib/usbstick.ts

```diff
@@ -0,0 +1,50 @@
+export const isAvailable = () => {
+  return !!window.kiosk
+}
+
+export const isPresent = async () => {
+  return isAvailable() && (await window.kiosk!.getUsbDrives()).length > 0
+}
+
+const getDevice = async () => {
```

Comment:
This function is not safe to call in non-kiosk browser environments, but the others are.